pkg/test/framework/components/istio/config.go

```diff
@@ -129,6 +130,10 @@ type Config struct {
 	// Do not wait for the validation webhook before completing the deployment. This is useful for
 	// doing deployments without Galley.
 	SkipWaitForValidationWebhook bool
+
+	// CustomSidecarInjectorNamespace allows injecting the sidecar from the specified namespace.
+	// if the value is "", use the default sidecar injection instead.
+	CustomSidecarInjectorNamespace string
```

Comment:
@howardjohn are you ok with this name? IIRC you wanted a simple flag `UseCustomSidecarInjector` (or something like that)

Reply:
well in the current state of the installer I think the injector must be in the same namespace as pilot, but there is no response reason for this. tbh the whole namespace configuration for the new installer makes no sense to me.

I think this is fine. I'm very close to integrating the installer directly into the test framework (just need to fix broken tests/bugs in installer) at which point I think we can remove these flags and look at helm values directly.